src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -499,5 +499,11 @@ public static unsafe float Int32BitsToSingle(int value)
 
             return *((float*)&value);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe short HalfToInt16Bits(Half value)
+        {
+            return *((short*)&value);
```

Comment:
Just wanted to call out it looks like the JIT does the right thing here: https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAMIR8AB14AbGFADKMgG68wMXAG4aNYgGYGuDFA5gMDABLZJAMxoBvGgwdNdsbABMIXSQE89AC2gmAPrAvBjqNAC+mrS63LjYljBMpAxCDHbUjk5+AQwAsuQAFOZWDL4AlOn2WY7EAOxlLMGh4ZmOUW0O1Q46OVAmeaTFFpZllRk1tQ0AVIWFuP790+UAZBUanQwdWd3ZC7l52sOlFVWbWfUMAKpc8YksAIK4ADwllmh9GAB8hbCj67sOhEgA===